pkg/csi/service/common/vsphereutil.go

```diff
@@ -564,11 +564,29 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 		},
 	}
 	// Query only the backing object details.
-	queryResult, err := manager.VolumeManager.QueryAllVolume(ctx, queryFilter, querySelection)
-	if err != nil {
-		log.Errorf("failed to call QueryVolume for volumeID: %q: %v", volumeID, err)
-		return false, err
+	var queryResult *cnstypes.CnsQueryResult
+	var err error
+	var queryAsyncNotSupported bool
+	if useAsyncQueryVolume {
```

Comment:
Can call `QueryVolumeUtil` from here too?

Reply:
Ah! Missed it. Thanks for catching this